charts/stable/wyoming-whisper/values.yaml

```diff
@@ -23,10 +24,12 @@ workload:
               port: 10300
               scheme: tcp
```

Comment:
btw the correct key for probes it `type`.

By default its `http` on readiness and liveness and `tcp` on startup.

(I know you didn't touch it on this PR, just mentioning in case its something that we should change)

Reply:
Good catch!

https://github.com/truecharts/charts/pull/11242